File: 6_Kyu/Sum of numbers/Sum_of_numbers.java
class Kata {

    public static long find_ways(double m, double n) {
        if (n == 0)
            return 0;
        else
            m = m + n - 1;
        n = n - 1;
        double c = 1;
        double b = 1;
        for (double i = 0; i < n; i++) {
            c = c * (m - i);
            b = b * (i + 1);
        }
        return (long) (c / b);
    }
}

File: 6_Kyu/Sum of numbers/Sum_of_numbers.py
def find_ways(m, n):
    if n == 0:
        return (0)
    else:
        m = m+n-1
        n = n-1
        c = 1
        b = 1
        for i in range(0, n):
            c = c*(m-i)
            b = b*(i+1)

        return(int(c/b))
